swift/schema.py

```diff
@@ -40,6 +40,10 @@ class Locatable(Element):
 class Comment(Locatable):
     text: string
 
+class Diagnostics(Locatable):
+    text: string
+    kind: int
```

Comment:
would it maybe make sense to use separate subclasses instead of `kind`, something like

```python
class Diagnostics(Locatable):
    text: string

class CompilerError(Diagnositcs):
    pass

class CompilerWarning(Diagnositcs):
    pass

class CompilerNotice(Diagnositcs):
    pass

class CompilerRemark(Diagnositcs):
    pass
```
to which we could later add our own
```python
class ExtractorError(Diagnostics):
    pass

class ExtractorWarning(Diagnostics):
    pass
```

On the other hand, this could also be done on the QL side.